content/install-user-css.js

```diff
@@ -0,0 +1,123 @@
+/* global runtimeSend */
+'use strict';
+
+function createSourceLoader() {
+  let source;
+
+  function fetchText(url) {
+    return new Promise((resolve, reject) => {
+      // you can't use fetch in Chrome under 'file:' protocol
+      const xhr = new XMLHttpRequest();
+      xhr.open('GET', url);
+      xhr.addEventListener('load', () => resolve(xhr.responseText));
+      xhr.addEventListener('error', () => reject(xhr));
+      xhr.send();
+    });
+  }
+
+  function load() {
+    return fetchText(location.href)
+      .then(newSource => {
+        source = newSource;
+        return source;
+      });
+  }
+
+  function watch(cb) {
+    let timer;
+    const DELAY = 1000;
+
+    function start() {
+      if (timer) {
+        return;
+      }
+      timer = setTimeout(check, DELAY);
+    }
+
+    function stop() {
+      clearTimeout(timer);
+      timer = null;
+    }
+
+    function check() {
+      fetchText(location.href)
+        .then(newSource => {
+          if (source !== newSource) {
+            source = newSource;
+            return cb(source);
+          }
+        })
+        .catch(console.log)
```

Comment:
It is used to suppress errors. Should we use a noop here? `() => {}`

Reply:
If those errors are critical show them to the user, otherwise continue to use console.log but use a translatable message e.g. `Stylus cannot download blabla, an error occured: ERROR`